server/core/Cargo.toml

```diff
@@ -15,9 +15,14 @@ repository = { workspace = true }
 test = true
 doctest = false
 
+[features]
+ui_htmx = []
```

Comment:
Probably either need to commit your changes or add this to `server/daemon/Cargo.toml` under its features:

```
ui_htmx = ["kanidmd_core/ui_htmx"]
```